website/docs/developer/research/research-deep-glacier-migration.md

```diff
@@ -0,0 +1,174 @@
+---
+id: research-deep-glacier-migration
+title: Deep archive storage migration Research Notes
+description: Research Notes on deep archive migration.
+---
+
+## Overview
+
+The goal of this wikipage is to provide an initial design and recommendation for migrating a customer's ORCA Flexible Retrieval (formerly Glacier) archive to DEEP ARCHIVE storage class. Deep archive S3 storage type is the cheapest glacier storage type currently present in AWS S3. For archive data that does not require immediate access such as backup or disaster recovery use cases, choosing `Deep Archive Access` tier could be a wise choice.
+- Storage cost is ~$1/TB data per month. 
+- Objects in the Deep Archive Access tier can be moved to the Frequent Access tier within 48 hours. Once the object is in the Frequent Access tier, users can send a GET request to retrieve the object. Like Flexible retrieval, the restored file is only available in frequent access for a configured number of days. For more information, see this [documentation](See https://docs.amazonaws.cn/en_us/AmazonS3/latest/userguide/restoring-objects-retrieval-options.html
+).
+
+### Assumptions
+
+- List of objects already exists in orca catalog and S3 bucket.
+- Users should be able to provide some information(filename) to retrieve objects possibly at the collection level.
+
+### Initial Implementation Idea
+
+A user might want to 
+- convert all the holdings or holdings with a known key prefix to DEEP ARCHIVE.
+- convert specific collection or granules to DEEP ARCHIVE which might not have the same prefix.
+
+#### Implementation idea for migrating all ORCA holdings to deep archive.
+
+- Use S3 lifecycle rule in this case possibly with a prefix option to migrate from glacier to deep archive.
```

Comment:
You are missing several items under here, you should talk about additional items like
- How does the ORCA catalog get updated with the new storage class value?
- What ORCA configuration changes should the end user be making (deployment variable/collection variable)?
- Do we expect this to be a manual effort with steps performed by the end user or should we automate this in some way?

A UML workflow diagram could help here to better understand the expected steps and outcome.

Reply:
- created [ORCA-503](https://bugs.earthdata.nasa.gov/browse/ORCA-503) this could possibly be a lambda to update the catalog
- we can add a required terraform variable that user will need to input that contains the list of granuleID/collection 
- yes we can automate this in terraform.